extensions/wikia/SpecialCategoryIntersection/SpecialCategoryIntersection.php

```diff
@@ -293,12 +289,13 @@ private function showFooter($out){
 		// Examples are now kept in wikitext so that each wiki can have its own examples if it wishes.
 		$exampleText = wfMsg('categoryintersection-footer-examples');
 		$rawExamples = explode("\n\n", $exampleText);
-		foreach($rawExamples as $singleExample){
+		$examples = array_map(function($singleExample) {
```

Comment:
There is no benefit to using `array_map` here, it is slower than the `foreach` loop and does not make the code more readable.